### ISSUE TITLE
Customize the theme

### ISSUE THREAD

**Issue by skostrigin** [lxqt/pcmanfm-qt#923]
Please add settings to customize the theme for pcmanfm-qt

**Comment by tsujan** [comment]
Doesn't make sense.

**Comment by agaida** [comment]
@tsujan - it makes sense and it is already implemented. Just run pcmanfm-qt outside a LXQt session ...

**Comment by tsujan** [comment]
> Just run pcmanfm-qt outside a LXQt session ...

I run it under KDE and E. If you mean *the icon theme*, it has a different story.

**Comment by agaida** [comment]
i mean the icon theme and i know the story

**Comment by tsujan** [comment]
Yes, the icon theme can be changed outside LXQt (its setting is made hidden inside LXQt).

Recently, pcmanfm-qt's icons became a mess under KDE. Although I use KDE only for testing, I created a desktop file with:
```
Exec=bash -c "export XDG_CURRENT_DESKTOP=LXQt && pcmanfm-qt %U"
```
and made it the default file manager. Now, its icons are OK.

Under Enlightenment, I set `QT_QPA_PLATFORMTHEME` to `lxqt` (with KDE, it can causes crashes in Plasma).

Anyhow, none of these is related to this report.
